### PR TITLE
Lan Bridge How To

### PR DESCRIPTION
**Pull request by marjohn56**
Instructions on how to setup a LAN Bridge

### CODE REVIEW

File: source/manual/how-tos/lan_bridge.rst [hunk 1]
```diff
@@ -0,0 +1,90 @@
+**How to set up a LAN Bridge**
+========================
+
+**Original Author:** Martin Wasley
```
Comment (fabianfrz) [inline source/manual/how-tos/lan_bridge.rst]: is that needed?

It is usually not in the docs but visible in the git logs…

**Comment by marjohn56** [comment]
Which bit.. the originally authored bit or the fact that the underline was too short, I corrected that after I loaded into my web server and saw the error. As for the authored by bit, when I didn't add it on the Zen doc, it was added for me.. :)

File: source/manual/how-tos/lan_bridge.rst (hunk 1 above)
Comment (fichtner) [reply source/manual/how-tos/lan_bridge.rst]: it's ok to put this in, some things can be imported from other articles or so and the docs are for everyone to share

Comment (fichtner) [reply source/manual/how-tos/lan_bridge.rst]: (feel free to update your own)

Comment (marjohn56) [reply source/manual/how-tos/lan_bridge.rst]: I'm quite happy not to have it, or I could use an alias.. 😋

**Comment by fichtner** [comment]
And merged, thanks.

**Pull request merged**